src/pages/login.tsx

```diff
@@ -2,24 +2,57 @@ import React, { useState } from "react";
 import type { NextPage } from "next";
 import styled from "@emotion/styled";
 import Link from "next/link";
+import Router from "next/router";
 
 const Login: NextPage = () => {
-  const [idCheck, setIdCheck] = useState("");
-  const [idError, setIdError] = useState(false);
-  const [pwCheck, setPwCheck] = useState("");
-  const [pwError, setPwError] = useState(false);
+  const [userId, setUserId] = useState("");
+  const [userPw, setUserPw] = useState("");
+  const [userIdPwError, setUserIdPwError] = useState(false);
```

Comment:
p3: join파일에는 타입을 다 적어주셨던데 이것도 명시해주면 좋을 것  같습니다.
```suggestion
  const [userId, setUserId] = useState("");
  const [userPw, setUserPw] = useState("");
  const [userIdPwError, setUserIdPwError] = useState(false);
```